courses/basic-algorithm-scripting/falsy-bouncer.js

```diff
@@ -0,0 +1,6 @@
+
+function bouncer(arr) {
+    return arr.filter( x => !!x );
```

Comment:
could you explain why you use !! ? 

PD: I don't want to say this is wrong or something, I'm just curious if you know the implications of use that operator.

Reply:
I use the first `not` operator to do an implicit conversion of the item to boolean and deny it, and the second one to deny it back so that the correct value gets returned. Examples:
**7**
`!7 -> false, !!7 -> true`
**''**
`!'' -> true, !!'' -> false`

Reply:
yep, but is more hard to read, bc you make me think the process (the __implicit conversion of the item to boolean and deny it__ part). 

For things that arent process/time sensitive, we can use the _Boolean()_ constructor: 

```js
Boolean(7) -> true
Boolean('') -> false
```
This beign said, if we have to do a lot of this operations, the use of `!!` its better bc its more faster. 

[source](https://www.sitepoint.com/javascript-double-negation-trick-trouble/)

Reply:
Given that this case is not process/time sensitive, I agree with using Boolean to make it more clear. Thank you! 🙂